### PR TITLE
Bootstrap datatables

### PR DESCRIPTION
**Pull request by ewels**
Made tasks table use bootstrap styling:

image

NB: Set to 10 items for the screengrab, but defaults to 25.

### CODE REVIEW

File: tower-web/src/styles.scss [hunk 1]
```diff
@@ -201,13 +198,6 @@ code {
...
     table-layout: fixed;
 
-    @media (min-width: 1400px) {
-      max-width: 1000px;
-    }
-    @media (min-width: 1650px) {
-      max-width: 1140px;
-    }
-
     th:nth-child(1) {
```
Comment (pditommaso) [inline tower-web/src/styles.scss]: I remember we needed this for #97. Let me check if it's still a problem.

**Comment by pditommaso** [comment]
Almost fine, but there's a wired misaligned in the header columns. Could not find why!

Screenshot 2019-12-03 at 11 42 11

Also, is it possible to have a black top and down border-line in the header? I find a bit less readable grey.

**Comment by ewels** [comment]
I can take a look. Which line do you mean sorry?

**Comment by pditommaso** [comment]
I mean the header top and bottom (horizontal) border. But the real issue is that the header cell are not (vertically) aligned with the data cells.

**Comment by pditommaso** [comment]
Awesome! The css width fixes the task table alignment issue. I've merged it manually. Instead, I've noticed the bootstrap datatables layout mess up the inner tables showed when opening the task details pane. 

Since it adds very little, I prefer to keep the table layout as it is. Thanks!

File: tower-web/src/styles.scss (hunk 1 above)
Comment (ewels) [reply tower-web/src/styles.scss]: Yes, now that the screen width is properly responsive and fluid after my first PR, I don't think these kinds of page overflow / width issues should be a problem any more.. 👍

**Comment by ewels** [comment]
Ok, nice! Glad I could help 😊 

Agreed that the existing style is totally sufficient. I personally prefer the consistency with the bootstrap styles, but I doubt anyone else will notice the difference and I am quite happy to not have to bugfix the column alignment issue 😅

**Pull request closed without merging**